docs/troubleshooting-guide/ldap.md

```diff
@@ -79,3 +80,8 @@ If you run into an issue, check that you can login to your LDAP directory and is
 
 [apache-directory-studio]: https://directory.apache.org/studio/
 [ldap-learn]: /learn/permissions/ldap-auth-access-control.html
+[ldap-docs]: ../administration-guide/10-single-sign-on.html#enabling-ldap-authentication
+
+## Are you still stuck?
+
+If you can’t solve your problem using the troubleshooting guides, search or ask the [Metabase community][discourse].
```

Comment:
Ooh good idea. I can make a note to include it on pages that deal with paid-only features. 

For LDAP, I think we should stick with the Discourse link to avoid hasty OSS readers from jumping to support (only to get re-directed to the forum again 😆 ).